test/multifile/synthesized-accessors/materialize-for-set-1/main.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %empty-directory(%t)
+
+// RUN: mkdir -p %t/onone %t/wmo
+// RUN: %target-build-swift -emit-module -emit-module-path %t/onone/library.swiftmodule -module-name=library -emit-library %S/library.swift
+// RUN: %target-build-swift %S/main.swift -I %t/onone/ -emit-ir > /dev/null
+
+// RUN: %target-build-swift -emit-module -emit-module-path %t/wmo/library.swiftmodule -module-name=library -emit-library -wmo %S/library.swift
+// RUN: %target-build-swift %S/main.swift -I %t/wmo/ -emit-ir > /dev/null
```

Comment:
I'm a little antsy about none of the output being checked here. Seems like this would work better as an IRGen test. There are also no calls to the variable's materializeForSet shown, either through the protocol or with a member of the class hierarchy as a static type.

Reply:
The crash was while deserializing the base class's materializeForSet, which happens when you define the override. But I can think of a way to make this more explicit.

Reply:
Yeah, instead of subclassing the subclass, the 'client' can just do an inout access on the property I guess.